configs/components/_base-ruby-selinux.rb

```diff
@@ -29,6 +29,13 @@
   pkg.sha256sum '77c294a927e6795c2e98f74b5c3adde9c8839690e9255b767c5fca6acff9b779'
   pkg.url "https://github.com/SELinuxProject/selinux/releases/download/#{pkg.get_version}/libselinux-#{pkg.get_version}.tar.gz"
   pkg.build_requires 'python3-distutils'
+elsif platform.name.start_with?('debian-13')
+  # SELinux 3.8.1 is the minimum version available in Debian 13 repos
```

Comment:
they already entered soft freeze on my birthday (yey?), so I doubt this will change.

Reply:
Good to know!

Reply:
And happy birthday! 🎉